code/game/objects/machinery/miner.dm

```diff
@@ -233,9 +221,11 @@
 		to_chat(user, "<span class='warning'>[src] is not ready to produce a shipment yet!</span>")
 		return
 
-	new mineral_produced(user.loc, stored_mineral)
-	stored_mineral -= 1
-	start_processing()
+	SSpoints.supply_points += mineral_value * stored_mineral
+	do_sparks(5, TRUE, src)
+	playsound(loc,'sound/effects/phasein.ogg', 50, FALSE)
+	visible_message("<span class='notice'>The [src] buzzes: Ore shipment has been sold for [mineral_value * stored_mineral] points.</span>")
```

Comment:
Visible message means xeno can see it and understand it AFAIK. Maybe use a say verb